src/app_rpc.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 Golioth, Inc.
+ * Copyright (c) 2023 Golioth, Inc.
```

Comment:
@MarkoPura just wanted to check: did you intend to update the copyright on all files in the app? if so, a couple were missed (e.g. `app_settings.h`, `app_dfu.h`, `Kconfig`, etc).

Also, more general question: do you guys want the copyright updated on all files in the app every year? or only on files that were changed in that year?

Reply:
I was also thinking about that and concluded we should update the files when/if they are changed, but I missed updating the `app_settings.h` file.
@szczys should I update all of the files now, and from now on we update the individual files only when they are changed?

Reply:
@szczys bump.
It would be good to merge this PR to Air Quality Monitor before release.

Reply:
Copyright date is cosmetic so I don't have opinion if you want to update it now or later. LGTM.

Reply:
@MarkoPura as there doesn't appear to be a [single right answer](https://reuse.software/faq/#years-copyright), here's my preference:

- When a new file is added, we add the year the file was created:
```
Copyright (c) 2022 Golioth, Inc.
```
- If a file is changed in subsequent years, we add the year of the latest change as a range, keeping the original year:
```
Copyright (c) 2022-2023 Golioth, Inc.
```

This seems to be the convention in Zephyr, for example: https://github.com/zephyrproject-rtos/zephyr/blob/5689916a70ad5c363bd7d5511b772ada90067d51/boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_common.dtsi#L2

BTW, having the year _at all_ has [questionable value](https://hynek.me/til/copyright-years/), so I think this is mainly a cosmetic thing, but I think following what Zephyr does is probably a safe bet.

If you're good with this, feel free to make the changes and get this merged.